db.h

```diff
@@ -119,7 +119,7 @@ class CAddrInfo {
   }
   int GetBanTime() const {
     if (IsGood()) return 0;
-    if (clientVersion && clientVersion < 31900) { return 604800; }
+    if (clientVersion && clientVersion < 180003) { return 604800; }
```

Comment:
Does it make sense to define this as a variable up top? I believe that would help maintenance in the future.